sympy/physics/control/lti.py

```diff
@@ -3838,6 +3840,108 @@ def num_outputs(self):
         """
         return self._D.rows
 
+    def state_vector(self, initial_conditions=None, input_vector=None, var=None):
+        r"""
+        Returns the State vector `x` given by the solution of the state equation
+            x'(t) = A * x(t) + B * u(t)
+
+        Parameters
+        ============
+
+        initial_conditions : Matrix
+            The initial conditions of `x` state vector.
+        input_vector : Matrix
```

Comment:
I don't see any test cases for when you pass something like `u = Matrix([Heaviside(t)])`. Does something like this work? It is super common to use a step function. What about `u = Matrix([sin(t)])`? Or any other common input function to test?

Reply:
@abhiphile It is your responsibility to make such tests. If you stick to MATLAB examples then only numerical tests will be supported and not symbolic ones. Let's make a separate PR only for such tests (it would cover all the `StateSpace` methods not just this one).

Reply:
Yes @faze-geek I'm working on a new PR which resolves the issues of this PR.